systemtest/src/test/java/io/strimzi/systemtest/backup/ColdBackupScriptST.java

```diff
@@ -0,0 +1,123 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.systemtest.backup;
+
+import static io.strimzi.systemtest.Constants.INTERNAL_CLIENTS_USED;
+import static io.strimzi.systemtest.Constants.REGRESSION;
+import static io.strimzi.test.TestUtils.USER_PATH;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.CoreMatchers.is;
+
+import java.util.Map;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+
+import io.strimzi.systemtest.AbstractST;
+import io.strimzi.systemtest.Constants;
+import io.strimzi.systemtest.kafkaclients.internalClients.InternalKafkaClient;
+import io.strimzi.systemtest.resources.ResourceManager;
+import io.strimzi.systemtest.resources.crd.KafkaClientsResource;
+import io.strimzi.systemtest.resources.crd.KafkaResource;
+import io.strimzi.systemtest.utils.kafkaUtils.KafkaUtils;
+import io.strimzi.systemtest.utils.kubeUtils.objects.NamespaceUtils;
+import io.strimzi.test.executor.Exec;
+
+@Tag(REGRESSION)
+@Tag(INTERNAL_CLIENTS_USED)
+public class ColdBackupScriptST extends AbstractST {
+
+    private static final Logger LOGGER = LogManager.getLogger(ColdBackupScriptST.class);
+    private static final String NAMESPACE = "cold-backup";
+
+    @Test
+    void backupAndRestore() {
+        String topicName = TOPIC_NAME;
```

Comment:
I guess you don't need this, just use `TOPIC_NAME` directly

Reply:
Yep.